pkg/cmd/admin/policy/reconcile_sccs.go

```diff
@@ -235,6 +236,10 @@ func (o *ReconcileSCCOptions) computeUpdatedSCC(expected kapi.SecurityContextCon
 		if actual.Priority != nil {
 			expected.Priority = actual.Priority
 		}
+
+		// preserve labels and annotations
+		util.AddObjectLabels(&expected, actual.Labels)
```

Comment:
this is just a merge of a map[string]string... don't need bespoke methods for each